noria-server/src/controller/sql/mod.rs

```diff
@@ -1002,16 +1006,21 @@ mod tests {
         use std::hash::{Hash, Hasher};
 
         let mut hasher = DefaultHasher::new();
-        for r in relations.iter() {
+        let mut r_vec: Vec<&str> = relations.to_vec();
+        r_vec.sort();  // QueryGraph.signature() sorts them, so we must to match
+        for r in &r_vec {
             r.hash(&mut hasher);
         }
-        for a in attrs.iter() {
+        let mut a_vec: Vec<&Column> = attrs.to_vec();
+        a_vec.sort();  // QueryGraph.signature() sorts them, so we must to match
+        for a in &a_vec {
             a.hash(&mut hasher);
         }
         for c in columns.iter() {
             OutputColumn::Data((*c).clone()).hash(&mut hasher);
         }
-        hasher.finish()
+        let temp = hasher.finish();
```

Comment:
Is this indirection via a temporary needed?

Reply:
Oops, I think I added that for printing purposes and then forgot to fully remove it when I deleted the print statement. Fixed.